tests_config/require_standalone.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const vm = require("vm");
+
+const sources = [
+  "standalone.js",
+  "parser-babylon.js",
+  "parser-flow.js",
+  "parser-typescript.js",
+  "parser-postcss.js",
+  "parser-graphql.js",
+  "parser-markdown.js",
+  "parser-vue.js"
+].map(filename =>
+  fs.readFileSync(path.join(process.env.PRETTIER_DIR, filename), "utf-8")
+);
+
+const sandbox = vm.createContext();
+vm.runInContext(sources.join(""), sandbox);
```

Comment:
You might want to join them with `;` to avoid ASI issues. I don't think our build output has any problems with that in practice, but better safe than sorry